ironfish/src/rpc/routes/eth/getBalance.ts

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Address } from '@ethereumjs/util'
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { FullNode } from '../../../node'
+import { BlockHeader } from '../../../primitives'
+import { RpcNotFoundError } from '../../adapters'
+import { ApiNamespace } from '../namespaces'
+import { registerEthRoute } from './ethRouter'
+
+export type EthGetBalanceRequest = {
+  address: string
+  blockNumber: string
```

Comment:
I was thinking it was a bit odd to use 'Number' since this might be a tag like 'latest', but I see that some other providers (e.g., QuickNode) call it `blockNumber`

Reply:
I agree, I am just following the eth spec, it is indeed a hex block sequence number I believe